timesketch/frontend/src/components/Sketch/TimelineListItem.vue

```diff
@@ -237,6 +266,26 @@ export default {
         }
         this.$store.dispatch('updateSketch', this.$store.state.sketch.id)
       }).catch((e) => {})
+    },
+    openFilteredTimeline: function (index, dataTypes) {
+      let searchQuery = ''
+      for (let i = 0; i < dataTypes.length; i++) {
+        const dt = dataTypes[i];
+        if (i != 0) {
+          searchQuery += ' OR '
+        }
+        searchQuery += 'data_type:"' + dt + '"'
+      }
+      router.push({name: 'SketchExplore', query: { index: index, q: searchQuery }})
```

Comment:
Use the router already present in the Vue instance:
`this.$router.push({name: 'SketchExplore', query: { index: index, q: searchQuery }})`

Reply:
Done